dev-docs/source/Testing/SANSGUI/SANSGUITests.rst

```diff
@@ -0,0 +1,76 @@
+.. _sans_gui_testing:
+
+SANS GUI Testing
+================
+
+.. contents::
+   :local:
+
+Data reduction
+--------------
+
+*Preparation*
+
+-  Get the ISIS sample data from the website
+-  Ensure that the ISIS sample data directory is in the search directories
+
+
+**Time required 20 - 30 minutes**
+
+--------------
+
+#. Open ``Interfaces`` > ``SANS`` > ``SANS v2``
+#. Set ``Instrument`` to ``LOQ``
```

Comment:
When PR #24128 merges, Instrument will no longer be setable, instead it is determined by the User File.

Reply:
Good spot Tom. Is that PR approved and ready to go? In which case let's just hold fire on this one until it's done.